api/application.py

```diff
@@ -23,7 +23,7 @@
 
 logging.basicConfig(level=logging.INFO)
 
-databaseConnectionString = Database.create_psql_connection_string(username=os.environ["DB_USERNAME"], password=os.environ["DB_PASSWORD"], host=os.environ["DB_HOST"], port=os.environ["DB_PORT"], name=os.environ["DB_NAME"])
+databaseConnectionString = Database.create_psql_connection_string(username=os.environ["REMOTE_DB_USERNAME"], password=os.environ["REMOTE_DB_PASSWORD"], host=os.environ["REMOTE_DB_HOST"], port=os.environ["REMOTE_DB_PORT"], name=os.environ["REMOTE_DB_NAME"])
```

Comment:
undo pls. you should just have a script that you can use to set the variables for you locally so you can map `export DB_USERNAME=$REOMTE_DB_USERNAME` etc when you need to instead of changing code (thats the point of envvars)